content/en/tracing/metrics/runtime_metrics/_index.md

```diff
@@ -1,17 +1,202 @@
+Runtime metrics cannot be enabled in code.
+
+Additional JMX metrics can be added using configuration files that are passed on using `dd.jmxfetch.config.dir` and `dd.jmxfetch.config`. You can also enable existing Datadog JMX integrations individually with the `dd.jmxfetch.<INTEGRATION_NAME>.enabled=true` parameter. This auto-embeds configuration from Datadog's existing JMX configuration files. See the [JMX Integration][100] for further details on configuration.
+
+[100]: /agent/configuration/agent-configuration-files/#agent-main-configuration-file
+{{< /programming-lang >}}
+
+{{< programming-lang lang="python" >}}
+If you are not using `ddtrace-run`, you can enable runtime metrics collection in code:
+
+```python
+from ddtrace.runtime import RuntimeMetrics
+RuntimeMetrics.enable()
+```
+{{< /programming-lang >}}
+
+{{< programming-lang lang="nodejs" >}}
+Runtime metrics collection can also be enabled in code with one configuration parameter in the tracing client through the tracer option: `tracer.init({ runtimeMetrics: true })`
+
+```js
+const tracer = require('dd-trace').init({
+  // ...
+  runtimeMetrics: true
+})
+```
+{{< /programming-lang >}}
+
+{{< programming-lang lang="go" >}}
+Runtime metrics collection can also be enabled in code by starting the tracer with the `WithRuntimeMetrics` option:
+
+```go
+tracer.Start(tracer.WithRuntimeMetrics())
+```
+
+If your Datadog Agent DogStatsD address differs from the default `localhost:8125`, use the [`WithDogstatsdAddress`][3] (or [`WithDogstatsdAddress` v2][9]) option (available starting in 1.18.0).
+
+[3]: https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer#WithDogstatsdAddress
+[9]: https://pkg.go.dev/github.com/DataDog/dd-trace-go/v2/ddtrace/tracer#WithDogstatsdAddress
+{{< /programming-lang >}}
+
+{{< programming-lang lang="ruby" >}}
+Runtime metrics collection can also be enabled in code by setting the following configuration in your Ruby application:
+
+```ruby
+# config/initializers/datadog.rb
+require 'datadog/statsd'
+require 'datadog' # Use 'ddtrace' if you're using v1.x
+
+Datadog.configure do |c|
+  c.runtime_metrics.enabled = true
+
+  # Optionally, you can configure the DogStatsD instance used for sending runtime metrics.
+  # DogStatsD is automatically configured with default settings if `dogstatsd-ruby` is available.
+  # You can configure with host and port of Datadog agent; defaults to 'localhost:8125'.
+  c.runtime_metrics.statsd = Datadog::Statsd.new
+end
+```
+{{< /programming-lang >}}
+
+{{< programming-lang lang="dotnet" >}}
+Runtime metrics cannot be enabled in code.
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
+
+## Data collected
+
+See [Runtime Metrics Data Collected][4] for the full list of collected runtime metrics.
+
+## View runtime metric dashboards
+
+After setup is complete, you can view your runtime metrics in:
+
+- The instrumented service's details page
+- The flame graph **Metrics** tab
+- Default runtime dashboards
```

Comment:
Food for thought - what if we have this link to [all runtime metrics dashboards](https://app.datadoghq.com/dashboard/lists/preset/3?q=runtime%20metrics&p=1)? It seems like the only things we'd need to do to rename it is update a few places from `JVM Metrics` to `JVM Runtime Metrics` so [built-in dashboard](https://github.com/DataDog/dogweb/blob/dfc17f6a73ae5059e536de3852f0526a16161a52/integration/java/apm_jvm_dashboard.json#L2) and [link to built-in dashboard](https://github.com/DataDog/web-ui/blob/0046f320b521350b3c5f75716cf090931554a08d/packages/apps/service-observability/lib/metrics/runtime-metrics.utils.ts#L14)

Reply:
I think that's a great idea. I'd say let's add the link once we make the corresponding changes to the JVM dashboards so we don't create any confusion for users